Source/bmalloc/bmalloc/IsoHeap.h

```diff
@@ -49,14 +49,43 @@ namespace api {
 // It's not valid to create an IsoHeap except in static storage.
 
 #if BUSE(LIBPAS)
-BEXPORT void* isoAllocate(pas_heap_ref& heapRef);
-BEXPORT void* isoTryAllocate(pas_heap_ref& heapRef);
+BEXPORT void* isoAllocate(pas_heap_ref&);
+BEXPORT void* isoTryAllocate(pas_heap_ref&);
+BEXPORT void* isoAllocateCompact(pas_heap_ref&);
+BEXPORT void* isoTryAllocateCompact(pas_heap_ref&);
 BEXPORT void isoDeallocate(void* ptr);
 
 // The name "LibPasBmallocHeapType" is important for the pas_status_reporter to work right.
 template<typename LibPasBmallocHeapType>
-struct IsoHeap {
-    constexpr IsoHeap(const char* = nullptr) { }
+struct IsoHeapBase {
+    constexpr IsoHeapBase(const char* = nullptr) { }
+
+    void scavenge()
+    {
+    }
```

Comment:
nit: You can also express this more compactly as:
```
void scavenge() { }
```

It enables more denser info at a glance especially since these are all empty or single statement functions.  Note: this single line style is not permitted for functions with multiple statements.